pyaedt/modules/Boundary.py

```diff
@@ -643,146 +847,140 @@ def override_component(
+class NativeComponentPCB(NativeComponentObject, object):
+    """Manages Native Component PCB data and execution.
 
-        Returns
-        -------
-        bool
-            ``True`` if successful. ``False`` otherwise.
-        """
-        # sanity check
-        valid_connectors = ["Solderbump", "Bondwire"]
-        if connector is not None and connector not in valid_connectors:
-            self._app.logger.error(
-                "{} option is not supported. Use one of the following: {}".format(
-                    connector, ", ".join(valid_connectors)
-                )
-            )
-            return False
-        solderbumps_map = {"Lumped": "SbLumped", "Cylinders": "SbCylinder", "Boxes": "SbBlock"}
-        for arg in [solderbumps_modeling, solderballs]:
-            if arg is not None and arg not in solderbumps_map:
-                self._app.logger.error(
-                    "{} option is not supported. Use one of the following: "
-                    "{}".format(arg, ", ".join(list(solderbumps_map.keys())))
-                )
-                return False
-        if bondwire_material not in self._app.materials.mat_names_aedt:
-            self._app.logger.error("{} material is not present in the library.".format(bondwire_material))
-            return False
+    Parameters
+    ----------
+    app : object
+        AEDT application from the ``pyaedt.application`` class.
+    component_type : str
+        Type of the component.
+    component_name : str
+        Name of the component.
+    props : dict
+        Properties of the boundary.
 
-        update_properties = {
-            "PartsChoice": 2,
-            "CreateTopSolderballs": connector is not None,
-            "TopConnectorType": connector,
-            "TopSolderballsModelType": solderbumps_map[solderbumps_modeling],
-            "BondwireMaterial": bondwire_material,
-            "BondwireDiameter": bondwire_diameter,
-            "CreateBottomSolderballs": solderballs is not None,
-            "BottomSolderballsModelType": solderbumps_map[solderballs],
-        }
+    Examples
+    --------
+    In this example, the returned object, ``par_beam`` is a ``pyaedt.modules.Boundary.NativeComponentObject`` instance.
+    >>> from pyaedt import Icepak
+    >>> ipk = Icepak(solution_type="SBR+")
+    >>> par_beam = ipk.create_ipk_3dcomponent_pcb()
+    """
 
-        self.props["NativeComponentDefinitionProvider"].update(update_properties)
-        return True
+    def __init__(self, app, component_type, component_name, props):
+        NativeComponentObject.__init__(self, app, component_type, component_name, props)
+
+    @property
+    def force_source_solve(self):
+        return self.props["NativeComponentDefinitionProvider"].get("DefnLink", {}).get("ForceSourceToSolve", False)
+
+    @force_source_solve.setter
+    def force_source_solve(self, val):
+        if not isinstance(val, bool):
+            self._app.logger.error("Only boolean can be accepted.")
```

Comment:
```suggestion
            self._app.logger.error("Only Boolean value can be accepted.")
```
Boolean is a proper noun.